src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Created.cs

```diff
@@ -101,7 +102,14 @@ public static void FileSystemWatcher_Created_Negative()
                 testFile.Flush();
 
                 // renaming a directory
-                testDir.Move(testDir.Path + "_rename");
+                //
+                // We don't do this on Linux because depending on the timing of MOVED_FROM and MOVED_TO events,
+                // a rename can trigger delete + create as a deliberate handling of an edge case, and this
+                // test is checking that no delete events are raised.
+                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                {
+                    testDir.Move(testDir.Path + "_rename");
+                }
 
                 // deleting a file & directory by leaving the using block
```

Comment:
Hmm, this will trigger a delete. I got overconfident and made the change to check delete instead of changed without re-testing. :(